tests/components/gree/test_bridge.py

```diff
@@ -38,13 +37,13 @@ async def test_discovery_after_setup(
     discovery.return_value.mock_devices = [mock_device_1, mock_device_2]
     device.side_effect = [mock_device_1, mock_device_2]
 
-    await async_setup_gree(hass)
+    entry = await async_setup_gree(hass)
     await hass.async_block_till_done()
 
     assert discovery.return_value.scan_count == 1
     assert len(hass.states.async_all(DOMAIN)) == 2
 
-    device_infos = [x.device.device_info for x in hass.data[GREE][COORDINATORS]]
+    device_infos = [x.device.device_info for x in entry.runtime_data]
```

Comment:
We should actually avoid touching these internals in the tests (not a blocker, just a remark)

Reply:
I didn't write these tests. Perhaps we can look into them in a future PR.

Reply:
Yea I just wanted to mention it